torch/nn/quantized/functional.py

```diff
@@ -360,6 +360,22 @@ def linear(input, weight, bias=None, scale=None, zero_point=None):
     _packed_params = torch.ops.quantized.linear_prepack(weight, bias)
     return torch.ops.quantized.linear(input, _packed_params, scale, zero_point)
 
+def max_pool1d(input, kernel_size, stride=None, padding=0, dilation=1,
```

Comment:
do we need this? i feel user won't be directly using these things in general

Reply:
This is to support the similarity to the `nn.functional`. Should I remove it?